postmaster/static/js/admins.js

```diff
@@ -279,6 +344,7 @@ function fillInTable () {
             html += '<td data-title="Username: "><a href="#" class="adminUsername" data-pk="' + item.id + '" data-url="/api/v1/admins/' + item.id + '" title="Click to change the username">' + filterText(item.username) + '</a></td>\
                     <td data-title="Password: "><a href="#" class="adminPassword" data-pk="' + item.id + '" data-url="/api/v1/admins/' + item.id + '" title="Click to change the password">●●●●●●●●</a></td>\
                     <td data-title="Name: "><a href="#" class="adminName" data-pk="' + item.id + '" data-url="/api/v1/admins/' + item.id + '" title="Click to change the name">' + filterText(item.name) + '</a></td>\
+                    <td data-title="2FA: ">' + (item.twoFactor ? ('<a href="#" class="admin2FADisable" data-pk="' + item.id + '" title="Click to disable 2-factor">Configured</a>') : ('<a href="#" class="admin2FAConfigure" data-pk="' + item.id + '" data-toggle="modal" data-target="#twoFAModal" title="Click to configure 2 Factor">Configure</a>')) + '</td>\
```

Comment:
I'd like to make this only show up in the row of the current admin. We can maybe add a field in the admin GET route that states whether or not this is the current admin that is logged in. That would give us the info we need to make the decision here.